### DISCUSSION TITLE
Is the “(OOB) Misclassification rate” not calculated by the confusion matrix？

### DISCUSSION THREAD

**Discussion by A-Pai** [kogalur/randomForestSRC#172] [Q&A]
rfsrc(Species~.,iris,importance=TRUE,perf.type="brier")
![image](https://user-images.githubusercontent.com/5112397/149288069-dbbb312d-4c4c-4fb5-9b16-18af9d0eccab.png)

**Comment by ishwaran** [comment]
It looks like the rate being reported is the normalized Brier score and not the misclassification error from the confusion matrix.  I will check, this could be a bug.
